docs/source/pythonapi/examples/tally-arithmetic.ipynb

```diff
@@ -418,29 +418,25 @@
     "\n",
     "# Instantiate flux Tally in moderator and fuel\n",
     "tally = openmc.Tally(name='flux')\n",
-    "tally.add_filter(openmc.Filter(type='cell', bins=[fuel_cell.id, moderator_cell.id]))\n",
-    "tally.add_filter(energy_filter)\n",
-    "tally.add_score('flux')\n",
+    "tally.filters = [openmc.Filter(type='cell', bins=[fuel_cell.id, moderator_cell.id]),\n",
+    "                 energy_filter]\n",
+    "tally.scores = ['flux']\n",
     "tallies_file.add_tally(tally)\n",
     "\n",
     "# Instantiate reaction rate Tally in fuel\n",
     "tally = openmc.Tally(name='fuel rxn rates')\n",
-    "tally.add_filter(openmc.Filter(type='cell', bins=[fuel_cell.id]))\n",
-    "tally.add_filter(energy_filter)\n",
-    "tally.add_score('nu-fission')\n",
-    "tally.add_score('scatter')\n",
-    "tally.add_nuclide(u238)\n",
-    "tally.add_nuclide(u235)\n",
+    "tally.filters = [openmc.Filter(type='cell', bins=[fuel_cell.id]),\n",
+    "                 energy_filter]\n",
```

Comment:
How about using `append` here on the continuation line as you do below? I think 1) it looks cleaner and 2) it illustrates that this can be done.